app/javascript/react/components/Map/Map.tsx

```diff
@@ -93,15 +91,14 @@ const Map = () => {
   const [selectedSessionId, setSelectedSessionId] = useState<number | null>(
     null
   );
-  const [selectedSessionType, setSelectedSessionType] = useState<SessionType>(
-    SessionTypes.FIXED
-  );
   const [selectedStreamId, setSelectedStreamId] = useState<number | null>(null);
 
+  // Selectors
+  const selectedSessionType = useAppSelector(selectSelectedSessionType);
```

Comment:
I didn't know we have `useAppSelector` 😅 Makes sense to use it everywhere for consistency but this is a note for me 🙈

Reply:
I didn't know either :D I've just noticed that so I decided to use :)